docs/tutorials.md

```diff
@@ -38,4 +38,8 @@ In this tutorial you will learn about the [DHIS2 CLI](https://cli.dhis2.nu/#/) a
 * Get an overview of the [DHIS2 Web API](https://youtu.be/_1bkoE-UKy0) in this short video presentation or check out the [DHIS2 Core Web API](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/introduction.html) documentation 
 
 
+### Learn about DHIS2 Components and Libraries 
+  
+* [Analytics](https://github.com/dhis2/analytics) - helper functions and common components for analytics applications
+* [SMS Compression & Syncing](https://github.com/dhis2/dhis2-android-capture-app/tree/master/docs/src/commonmark/en/content/tech-guides) - how the SMS library works and how it is used in server and Android side
```

Comment:
This is probably less of a tutorial and also is specific to Android, so maybe should be in a separate section?  Where do we want to put Android content?  @deboragaleano we should set up a "point person" for Android documentation from the Android team to ask these questions and contribute to the dev portal.

Reply:
@amcgee Thanks Austin for reviewing this tutorial! I agree with you, I added that section from the old portal without really having a dedicated place for it - I'll comment it out for now. 

I was thinking that maybe the Android content can have it's own "Docs" page same as Web (which is now the official "Docs" page but we can change that to something more specific to differentiate?) 

In any case, I can ask Jose to see what he suggests and set up a meeting to discuss?

Reply:
Sounds good, thanks!